src/meta/app.ts

```diff
@@ -37,13 +37,19 @@ export enum AssessmentHomeRouteNames {
   links = 'links',
 }
 
+export enum AdminRouteNames {
+  userManagement = 'userManagement',
+  dataExport = 'dataExport',
```

Comment:
is dataExport still used?

Reply:
idk, I can comment it for now, as the section value in Admin.tsx